nifi-commons/nifi-property-utils/src/main/java/org/apache/nifi/properties/ApplicationProperties.java

```diff
@@ -22,7 +22,7 @@
 /**
  * A tagging class that represents the main configuration properties for an application (e.g. NiFi or NiFi Registry).
  */
-public class ApplicationProperties extends StandardReadableProperties {
+public abstract class ApplicationProperties extends StandardReadableProperties {
```

Comment:
Is there a reason for marking this class as `abstract` as part of this PR?

Reply:
No, this was a left over artifact of something that I backed out -- will remove.